adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -1,846 +0,0 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
@piqiums This class should not have been deleted - it is part of the public API surface.

We'll likely have to revert this PR, can sync in scrum